regtests/polaris-reg-test

```diff
@@ -15,13 +15,21 @@
 # limitations under the License.
 #
 
-################################################
-# This is a modified copy of the script in the #
-# parent directory, used for regression tests. #
-################################################
+####################################################
+# This is a modified copy of the polaris script in #
+# the parent directory, used for regression tests. #
+####################################################
```

Comment:
Taking a close look at two files, `regtests/polaris-reg-test` and `polaris`. looks like they are same semantically. Basically it does the following steps:
1. save the current directory 
2. go to the `$SCRIPT_DI`
3. do something
4. back to the current directory

In that case, why cannot we just use one file instead of having two files? Am I missing something? cc @eric-maynard

Reply:
I was under assumption this is due to the reasoning I posted above where the Dockerfile was in `regtests` dir and if we only have the script on the parent dir, we will need to set context to be on the project's root dir then copy files into the container that is relative to the project's root path.

Then the change back to the change dir thing, I think they are the same...one is using variable to keep the original location then switched back and the other is using pushd/popd for those.

Reply:
> I was under assumption this is due to the reasoning I posted above where the Dockerfile was in regtests dir and if we only have the script on the parent dir, we will need to set context to be on the project's root dir then copy files into the container that is relative to the project's root path.

I think you are right. There are multiple solutions, e.g., moving the dockerfile out of dir regtests, and change its name. It may be too much ask in this PR. We can keep the current one, but I'd keep the file content of `regtests/polaris-reg-test` exactly same as the `polaris` in the root dir, and remove the duplication in a followup PR.

Reply:
The content shouldn't be the same as they need to use different paths